.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
I think you need to update this slightly to have different configuration for the yml files which do not support tabs for indentation. See: https://yaml.org/faq.html

You can specify in the .editorconfig file that all .yaml and .yml files use spaces with a 4 spaces, for example. See: https://editorconfig.org